Cataclysm/APLs/RogueCombat.simc

```diff
@@ -3,8 +3,8 @@ Smufriks built form wowtbc logic
 actions+=/killing_spree,if=energy<35&buff.adrenaline_rush.down
 actions+=/adrenaline_rush,if=energy<20
 actions+=/slice_and_dice,if=buff.slice_and_dice.remains<2
-actions+=/rupture,if=combo_points=5&!ticking
-actions+=/revealing_strike,if=combo_points = 4 & buff.revealing_strike.down & buff.slice_and_dice.remains > 5
+actions+=/rupture,if=combo_points=5&!ticking&(boss|active_enemies==1)&!buff.deep_insight.up&!buff.bladeflurry.up&!buff.adrenaline_rush.up&time_to_die>12&!buff.bloodlust.up
```

Comment:
Should be single equal sign (`active_enemies=1`).
Should be `buff.blade_flurry.up` (with underscore).